src/Page/Form.purs

```diff
@@ -184,8 +184,8 @@ component =
   renderField :: Field ExprType -> H.ComponentHTML Query
   renderField (field) =
     FormField.field_
-      { label: HH.text $ show field.name
-      , helpText: Just $ show field.description
+      { label: HH.text $ print field.name
```

Comment:
😻 Thanks for doing this. It's a small change, but I feel like it's enough to make things look nicer.